java/src/main/java/com/saucelabs/saucebindings/DataCenter.java

```diff
@@ -3,9 +3,11 @@
 import lombok.Getter;
 
 public enum DataCenter {
-    US_WEST("ondemand.us-west-1.saucelabs.com"),
-    US_EAST("ondemand.us-east-1.saucelabs.com"),
-    EU_CENTRAL("ondemand.eu-central-1.saucelabs.com");
+    US_LEGACY("https://ondemand.saucelabs.com/wd/hub"),
```

Comment:
@titusfortner what's the use case for this?

Reply:
We'll see if we need it, but mobile + w3c + us-west/us-east wasn't working, so I was setting that to default.